src/daos/docker/test.sh

```diff
@@ -13,11 +13,11 @@ set -x
 source $CEPH_PATH/src/daos/error_handler.sh
 source $CEPH_PATH/src/daos/set_boolean.sh
 source $CEPH_PATH/src/daos/require_variables.sh
-source $CEPH_PATH/src/daos/daos_format.sh
-source $CEPH_PATH/src/daos/daos_pool_create.sh
-source $CEPH_PATH/src/daos/radosgw_start.sh
-source $CEPH_PATH/src/daos/radosgw_stop.sh
-source $CEPH_PATH/src/daos/radosgw_create_s3bucket.sh
+source $CEPH_PATH/src/daos/daos/daos_format.sh
+source $CEPH_PATH/src/daos/daos/daos_pool_create.sh
+source $CEPH_PATH/src/daos/radosgw/radosgw/radosgw_start.sh
+source $CEPH_PATH/src/daos/radosgw/radosgw/radosgw_stop.sh
```

Comment:
Are these paths correct? There is no `src/daos/radosgw/radosgw`

Reply:
already fixed

Reply:
The latest version of the code does not fix them

Reply:
I created another branch, squashed & merged because I had already deleted the branch.  Its indeed fixed.  The jenkins build is working: https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/sv_space/job/rgwdaos-test/102/console